bugsnag-android-core/src/test/java/com/bugsnag/android/RemoteConfigSerializationTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.internal.DateUtils
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameter
+import org.junit.runners.Parameterized.Parameters
+
+@RunWith(Parameterized::class)
+internal class RemoteConfigSerializationTest {
+
+    companion object {
+        val allRule: DiscardRule = DiscardRule.All()
+        val allHandledRule: DiscardRule = DiscardRule.AllHandled()
+        val dateString = "2024-01-15T10:30:45.123Z"
+        val date = DateUtils.fromIso8601(dateString)
```

Comment:
Not sure that these really add anything to the readability of the test. I think they can reasonably inlined